cli/smartcontract/smart_contract.go

```diff
@@ -214,16 +218,42 @@ func NewCommands() []cli.Command {
     * 'string\:string' is a string with a value of 'string:string'
     * '03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c' is a
       key with a value of '03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c'
+
+   Cosigners represent a set of Uint160 hashes with witness scopes and are used
+   to verify hashes in System.Runtime.CheckWitness syscall. To specify cosigners
+   use [cosigner]:[scope] syntax were
+    * 'cosigner' is hex-encoded 160 bit (20 byte) LE value of cosigner's address,
+                 which could have '0x' preffix.
+    * 'scope' is a set of cosigner's scopes, which could be:
+        - 'Global' - allows this witness in all contexts. This cannot be combined
+                     with other flags.
+        - 'CalledByEntry' - means that this condition must hold: EntryScriptHash 
+                            == CallingScriptHash. The witness/permission/signature
+                            given on first invocation will automatically expire if
+                            entering deeper internal invokes. This can be default
+                            safe choice for native NEO/GAS.
+        - 'CustomContracts' - define custom hash for contract-specific.
+        - 'CustomGroups' - define custom pubkey for group members.
+
+   If no scopes were specified, 'Global' used as default. If no cosigners were
+   specified, an empty array will be passed.
+
+   Examples:
+    * '0000000009070e030d0f0e020d0c06050e030c02'
+    * '0x0000000009070e030d0f0e020d0c06050e030c02'
+    * '0x0000000009070e030d0f0e020d0c06050e030c02:Global'
+    * '0000000009070e030d0f0e020d0c06050e030c02:CalledByEntry,CustomGroups'   
 `,
 				Action: testInvokeFunction,
 				Flags: []cli.Flag{
 					endpointFlag,
 				},
 			},
 			{
-				Name:   "testinvokescript",
-				Usage:  "Invoke compiled AVM code on the blockchain (test mode, not creating a transaction for it)",
-				Action: testInvokeScript,
+				Name:      "testinvokescript",
+				Usage:     "Invoke compiled AVM code on the blockchain (test mode, not creating a transaction for it)",
+				UsageText: "neo-go contract testinvokescript -e endpoint -i input.avm [cosigners...]",
```

Comment:
A description with a reference to `testinvokefunction` for `cosigners` would be helpful.